lib/elixir/lib/kernel/parallel_compiler.ex

```diff
@@ -464,6 +472,7 @@ defmodule Kernel.ParallelCompiler do
   defp terminate(queued) do
     for {pid, _, _, _} <- queued do
       Process.exit(pid, :kill)
+      discard_down(pid)
```

Comment:
Let's do this in a separate process so we send and handle all kills concurrently. So it would be:

```elixir
for {pid, _, _, _} <- queued, do: Process.exit(pid, :kill)
for {pid, _, _, _} <- queued, do: discard_down(pid)
```

Reply:
👍

Actually, that was original implementation, but I changed after seeing [this PR](https://github.com/elixir-lang/elixir/pull/8239/files#diff-e1812f5a5c51b7f59c6bcdf0040eb7cbR561). 

Should I open another PR and change also that one?

Reply:
Ys, I would change the other one too. :)

Reply:
Done https://github.com/elixir-lang/elixir/pull/8260